src/main/java/duke/command/BlahCommand.java

```diff
@@ -0,0 +1,22 @@
+package duke.command;
+
+import duke.Ui;
+import duke.Storage;
+import duke.tasks.TaskList;
+
+public class BlahCommand extends Command {
+    String command;
+    public BlahCommand(String command) {
+        this.command = command;
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
I think you might have missed out on the header comments for the Command classes haha but remember to include them for all public methods :-)